src/common/components/loading/loading.tpl.html

```diff
@@ -1,16 +1,16 @@
 <div ng-class="{
-        'loading-centered loading-overlay' : $ctrl.type === 'overlay',
-        'loading-centered' : $ctrl.type === 'centered'
+        'loading-centered': $ctrl.type === 'centered' || $ctrl.type === 'overlay',
+        'loading-overlay': $ctrl.type === 'overlay'
       }">
   <div>
     <span ng-class="{ 'mr-' : $ctrl.inline === 'true', 'center-block' : $ctrl.inline !== 'true'  }">
       <ng-transclude></ng-transclude>
     </span>
-    <span class="loading-wrap">
-      <span class="loading loading1"></span>
-      <span class="loading loading2"></span>
-      <span class="loading loading3"></span>
-      <span class="loading loading4"></span>
+    <span ng-class="loading-wrap">
```

Comment:
Good catch. I guess I left that from my experiment with CSS Modules where you had to import the renamed class name.